app/viewmodels/helpers/messages/ViewMessageHelper.scala

```diff
@@ -36,34 +36,20 @@ class ViewMessageHelper @Inject()(
                                    list: list,
                                    link: link,
                                    p: p,
-                                   govukTable: GovukTable) extends DateUtils with TagFluency {
+                                   summary_list: summary_list,
+                                   h2: h2) extends DateUtils with TagFluency {
 
   private[viewmodels] def dataRows(message: Message)
-                                  (implicit messages: Messages): Seq[Seq[TableRow]] = {
-    Seq(
-      Seq(
-        TableRow(content = Text(messages("viewMessage.table.messageType.label"))),
-        TableRow(content = Text(messages(messagesHelper.formattedMessageType(message))))
-      ),
-      Seq(
-        TableRow(content = Text(messages("viewMessage.table.arc.label"))),
-        TableRow(content = Text(message.arc.getOrElse("")))
-      ),
-      Seq(
-        TableRow(content = Text(messages("viewMessage.table.lrn.label"))),
-        TableRow(content = Text(message.lrn.getOrElse("")))
-      )
-    )
+                                  (implicit messages: Messages): Html = {
+    val optMessageTypeRow = if(!message.isAnErrorMessage) Seq(messages("viewMessage.table.messageType.label") -> messages(messagesHelper.formattedMessageType(message))) else Seq()
```

Comment:
Is there a reason `dataRows` is in a separate method given it's the only line in `constructMovementInformation`?

Reply:
No, there isn't removing